lib/pbench/agent/tool_meister.py

```diff
@@ -638,11 +641,12 @@ class ToolMeister:
     Meister is running remotely, then it will use a local temporary directory
     to write it's data, and will send that data to the Tool Data Sink during
     the "send" phase.
-
     """
 
     @staticmethod
-    def fetch_params(params):
+    def fetch_params(
+        params: dict,
+    ) -> (str, str, str, str, str, str, str, str, dict, str):
```

Comment:
I've thought to comment on this before, but never did. This type hint (which I believe may actually be a problem, as previously noted), would be a lot easier to read and use as a `NamedTuple`,

```
class ParamResponse(Namedtuple):
    benchmark_run_dir: str
    channel_prefix: str
    ...
```